### PR TITLE
Create pull_request_template.md

### PR DESCRIPTION
**Pull request by tomschr**
This is a first attempt to make PR for susedoc.github.io easier.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Changes
+<!-- describe your changes briefly -->
+
+* [ ] I also added the config to docserv-config
```
Comment (sknorr) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:4]: Hm, I am not sure about the wording here. E.g. the `main` branches are usually not on docs.suse.com ... i.e.: there does not need to be a change in every case.

Not sure whether this proposal is better or worse:

```suggestion
* [ ] This change affects documentation already published on docs.suse.com as well
    * [ ] I made an MR to the `docserv-config` repo
    * [ ] Please update the `docserv-config` repo for me
```

**Review by sknorr — COMMENTED** [review]
coolio

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -0,0 +1,5 @@
+# https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners
+
+# Auto-assign Liam on all incoming PRs
```
[the remark this answers is not in the document]
Comment (tomschr) [reply .github/CODEOWNERS:3]: Ahh right. Good catch! :+1:

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 1 above)
Comment (tomschr) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:4]: Yeah, it was a bad, quick hack before the next meeting. I take your suggestion.

**Pull request merged**